src/browser_action/popup.html

```diff
@@ -105,12 +105,21 @@ <h1>Metrics</h1>
         <span id="mobile-warning">
           Mobile performance may be significantly slower.
           <a href="https://web.dev/load-fast-enough-for-pwa/" target="_blank" rel="noopener">Learn more</a>
         </span>
+        <span id="settings-link">
+          <a href="/src/options/options.html" target="_blank">
+            <svg width="20" height="20" viewBox="0 0 20 20" role="img" fill="currentColor" xmlns="http://www.w3.org/2000/svg" aria-labelledby="#settings-icon">
+              <title id="settings-icon">Settings</title>
+              <path fill-rule="evenodd" clip-rule="evenodd" d="M8.93528 16.0938H11.0741L11.3266 14.3076L11.9527 13.9705C12.1186 13.8812 12.2727 13.7917 12.4209 13.6949L13.0333 13.2945L14.7536 13.9826L15.8161 12.1432L14.3513 10.9964L14.4008 10.2542C14.4085 10.1388 14.4123 10.0742 14.4123 10.011C14.4123 9.94773 14.4085 9.8832 14.4008 9.76778L14.3513 9.02553L15.8167 7.87825L14.7515 6.03729L13.026 6.7275L12.4135 6.32706C12.2664 6.23087 12.1173 6.14462 11.9575 6.06173L11.315 5.7286L11.0547 3.90625H8.93363L8.67804 5.71432L8.05188 6.05148C7.88598 6.14081 7.73184 6.23022 7.58373 6.32706L6.96916 6.7289L5.24869 6.03501L4.1904 7.85985L5.66064 9.01089L5.61115 9.75313C5.6036 9.86638 5.59963 9.9396 5.59963 9.99634C5.59963 10.0596 5.60346 10.1241 5.61115 10.2395L5.66083 10.9846L4.18814 12.1301L5.24465 13.9602L6.96397 13.2725L7.57641 13.6729C7.72352 13.7691 7.87263 13.8554 8.03249 13.9383L8.67494 14.2714L8.93528 16.0938ZM5.48913 15.377C4.97669 15.5674 4.413 15.3551 4.1641 14.9012L2.82442 12.5805C2.5682 12.0974 2.67801 11.5264 3.08797 11.2042L4.20802 10.3331C4.2007 10.2233 4.19338 10.1135 4.19338 9.99634C4.19338 9.88653 4.2007 9.7694 4.20802 9.65959L3.09529 8.78843C2.66337 8.459 2.55356 7.86603 2.82442 7.41215L4.17874 5.07687C4.42764 4.62299 4.99133 4.41801 5.48913 4.61567L6.81416 5.15007C7.0045 5.02562 7.19484 4.91581 7.38517 4.81332L7.58283 3.41508C7.64871 2.90264 8.10259 2.5 8.637 2.5H11.3456C11.8874 2.5 12.3412 2.89531 12.4071 3.42972L12.6048 4.81332C12.8024 4.91581 12.9928 5.02562 13.1831 5.15007L14.5008 4.62299C15.0206 4.43265 15.5843 4.64495 15.8332 5.09883L17.1802 7.42679C17.4437 7.90996 17.3266 8.48097 16.9166 8.80307L15.8039 9.67423C15.8112 9.78404 15.8185 9.89385 15.8185 10.011C15.8185 10.1281 15.8112 10.2379 15.8039 10.3477L16.9166 11.2189C17.3266 11.5483 17.4437 12.1193 17.1875 12.5805L15.8259 14.9378C15.5769 15.3917 15.0133 15.5966 14.5081 15.399L13.1904 14.8719C13.0001 14.9963 12.8098 15.1061 12.6194 15.2086L12.4218 16.6069C12.3486 17.1047 11.8947 17.5 11.353 17.5H8.64432C8.10259 17.5 7.64871 17.1047 7.58283 16.5703L7.38517 15.1867C7.18752 15.0842 6.99718 14.9744 6.80684 14.8499L5.48913 15.377Z" fill="currentColor"/>
+              <path d="M9.9988 12.5585C11.4139 12.5585 12.561 11.4114 12.561 9.99631C12.561 8.58123 11.4139 7.43408 9.9988 7.43408C8.58373 7.43408 7.43658 8.58123 7.43658 9.99631C7.43658 11.4114 8.58373 12.5585 9.9988 12.5585Z" fill="currentColor"/>
+            </svg>
```

Comment:
You may already know this, but passing through [SVGOMG](https://jakearchibald.github.io/svgomg/) can usually make inline SVG considerably smaller with no perceptual loss

Reply:
Stole this from DevTools source to presumed it was optomised already! But SVGOMG managed to shave it down a little more.